hstrat/_auxiliary_lib/_import_cpp_impls.py

```diff
@@ -0,0 +1,78 @@
+import importlib
+import logging
+import os
+import types
+
+import opytional as opyt
+
+from ._except_wrap import except_wrap
+from ._is_in_unit_test import is_in_unit_test
+
+
+def _import_cppimport(module_name: str, package: str) -> types.ModuleType:
+    """Implementation detail for import_cpp_impls"""
+    import cppimport
+
+    return cppimport.imp(f"{package}.{module_name}")
+
+
+def _import_importlib(module_name: str, package: str) -> types.ModuleType:
+    """Implementation detail for import_cpp_impls"""
+    return importlib.import_module(f".{module_name}", package=package)
+
+
+def import_cpp_impls(module_name: str, package: str) -> types.ModuleType:
```

Comment:
maybe a slightly more explicit name ? something having to do with autodelegating import or maybe using cppimport